src/collections/mapContains/MapContains1.kts

```diff
@@ -0,0 +1,21 @@
+package collections.mapContains
+
+import java.util.concurrent.ConcurrentHashMap
+
+val scores = ConcurrentHashMap<String, Int>()
+scores["Steve"] = 9
+scores["Rachel"] = 11
+
+val scoresHigherThan10 = scores.filterValues { it > 10 }
+
+scores.contains("Steve")
+scores.contains("Rachel")
+
+scoresHigherThan10.contains("Steve")
+scoresHigherThan10.contains("Rachel")
+
+// What will the results be?
+// a) true true true true
+// b) false false false false
```

Comment:
Maybe this one should be replaced with `true true false false` to represent the case where the audience expects that `scoresHigherThan10` is empty.